7.7.Поиск_ошибок_и_ревью_кода/7.7.3.py

```diff
@@ -0,0 +1,13 @@
+count = 0
+p = 1
+
+for i in range(1, 11):  
+    x = int(input())
+    if x > -1:  
+        p = p * x
+        count += 1
+if count > 0:
+    print(count)
+    print(p)
```

Comment:
Используй один вызов функции `print()`:
```suggestion
    print(count, p, sep="\n")
```